surfman/Cargo.toml

```diff
@@ -36,7 +36,7 @@ log = "0.4"
 parking_lot = "0.10.2"
 
 [dependencies.euclid]
-version = "0.20"
+version = ">=0.20,<=0.22"
```

Comment:
Why the range?

Reply:
Because there's a lot of other stuff to update basically :-)